core/src/main/scala/org/apache/spark/ui/WebUI.scala

```diff
@@ -95,6 +101,16 @@ private[spark] abstract class WebUI(
     serverInfo.foreach(_.addHandler(handler, securityManager))
   }
 
+  /** Attaches a handler to this UI. */
+  def attachNewHandler(contextPath: String,
```

Comment:
This doesn't follow Spark's multi-line argument style. Also in all your other methods.

For this particular method, the `New` in the name is unnecessary. It just raises questions about why this handler is "new" and the others aren't.

Reply:
Done. Changed Formatting and Method name